ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -185,6 +191,9 @@ export const styles = reactStyles({
 
 const CREATE_BILLING_ACCOUNT_OPTION_VALUE = 'CREATE_BILLING_ACCOUNT_OPTION';
 
+// default to creating workspaces in the Registered Tier
```

Comment:
Note: Registered is still the ONLY tier accessible from the UI after this change.  This PR is only part of the journey.